examples/app/pivot/main.py

```diff
@@ -0,0 +1,377 @@
+from __future__ import division
+import os
+import math
+import json
+import numpy as np
+import pandas as pd
+import collections
+import bokeh.io as bio
+import bokeh.layouts as bl
+import bokeh.models as bm
+import bokeh.models.widgets as bmw
+import bokeh.models.sources as bms
+import bokeh.models.tools as bmt
+import bokeh.plotting as bp
+import datetime
+import six.moves.urllib.parse as urlp
+
+PLOT_WIDTH = 300
+PLOT_HEIGHT = 300
+PLOT_FONT_SIZE = 10
+PLOT_AXIS_LABEL_SIZE = 8
+PLOT_LABEL_ORIENTATION = 45
```

Comment:
These variables are just used as defaults for the widgets. But the styling itself is coming from the widget values, not directly from these variables. Let me know if you have another idea of how to do this. In the meantime, I've removed theme.yaml.